reference/tools/system/package_manager.rst

```diff
@@ -144,7 +144,7 @@ conan.tools.system.package_manager.Apt
 --------------------------------------
 
 Will invoke the *apt-get* command. Enabled by default for **Linux** with distribution
-names: *ubuntu* and *debian*.
+names: *ubuntu*, *debian*, *raspberry* and *linuxmint*.
```

Comment:
```suggestion
names: *ubuntu*, *debian*, *raspbian* and *linuxmint*.
```

Reply:
Did I really commit "raspberry"? 😬
Anyways, thank you for catching that and all your help!